src/compas_timber/utils/__init__.py

```diff
@@ -190,6 +190,12 @@ def distance_segment_segment(segment_a, segment_b):
 
     """
     pta, ptb = intersection_line_line(segment_a, segment_b)
+    if not pta:  # segments are parallel
+        dists = []
+        for pair in [(segment_a[0], segment_b[0]), (segment_a[1], segment_b[0]), (segment_b[0], segment_a[1]), (segment_b[1], segment_a[1])]:
+            """gets shortest distance between all 4 possible pairs of endpoints. only for L/I_Topo. T_Topology cannot have paralell segments"""
```

Comment:
please use `#` for in-code comments

Reply:
if possible, try to use something from `https://docs.python.org/3/library/itertools.html` here instead. would make for a more readable code.